CMakeLists.txt

```diff
@@ -92,6 +92,13 @@ if( HAVE_SERIALBOX )
 endif()
 
 ecbuild_find_package( NAME loki )
+ecbuild_find_python(VERSION 3.8 NO_LIBS)
+
+if ( PYTHON_EXECUTABLE )
+  execute_process(COMMAND ${PYTHON_EXECUTABLE} --version)
+  execute_process(COMMAND ${PYTHON_EXECUTABLE} -m pip install --upgrade pip)
+  execute_process(COMMAND ${PYTHON_EXECUTABLE} -m pip install f90wrap h5py)
```

Comment:
We should really pip install the utility Python package here, and let pip figure out that `f90wrap` and `h5py` are the downstream dependencies.

Reply:
But to create the package one needs to have f90wrap already installed ... unless one creates a stub package with dependencies first, perhaps, and then progress with a real one ?

Reply:
Yes, but that is the point of having pip: if your `cloudsc_pyiface` Python package (should really use consistent naming!) specifies these as dependencies, then pip will install all dependencies first before installing the package itself. It will also detect if they are already installed and attempt proper dependency resolution if there are complex dependency chains or if things already exist in a given venv.

Reply:
I  get it, but the pip install can be executed on the package that already exists. Before f90wrap/f2py is executed on the Fortran pieces, the thing we want to install doesn't exist, correct ?

Reply:
It does though. I'm trying to install the data utilities and all the boilerplate stuff, so that we can then get rid of the second pip install at make time. Pip and the Python package dependency resolution should only be executed during configure. You have dynamic module loading in the driver already, so as long as we make sure that that is guarded enough, there is no need to force the second pip install.